src/Adaptivity/RefinementRules.jl

```diff
@@ -104,6 +104,92 @@ function bundle_points_by_subcell(rr::RefinementRule,x::AbstractArray{<:Point})
   return Table(data,ptrs)
 end
 
+
+# Faces to child faces, dof maps
+
+# [Face dimension][Coarse Face id] -> [Fine faces]
+function get_d_to_face_to_child_faces(rr::RefinementRule) 
+  get_d_to_face_to_child_faces(rr,RefinementRuleType(rr))
+end
+
+function get_d_to_face_to_child_faces(::RefinementRule,::RefinementRuleType)
+  @notimplemented
+end
+
+function _get_terms(poly::Polytope,orders)
+  _nodes, facenodes = ReferenceFEs._compute_nodes(poly,orders)
+  terms = ReferenceFEs._coords_to_terms(_nodes,orders)
+  return terms
+end
+
+function _get_face_orders(p::Polytope{Dc},D::Int,orders::Tuple) where Dc
+  @check length(orders) == Dc
+  @check 1 <= D < Dc
+  @check is_n_cube(p)
+
+  if D == 1 # Edges (2D, 3D)
+    tangents = get_edge_tangent(p)
+    face_orders = map(tangents) do t
+      axis = findfirst(i -> abs(t[i]) > 0.5 ,1:Dc)
+      return [orders[axis]]
+    end
+  elseif D == Dc-1 # Faces (3D)
+    normals = get_facet_normal(p)
+    face_orders = map(normals) do n
+      mask = map(i -> abs(n[i]) < 1.e-3,1:Dc)
+      return [orders[mask]...]
+    end
+  else
+    @notimplemented
+  end
+
+  return face_orders
+end
+
+function coarse_nodes_above_fine_nodes(rr::RefinementRule{ExtrusionPolytope{Dc}},
```

Comment:
wouldnt be a name such as:

`get_face_subface_ldof_to_cell_ldof` 

more appropriate? just a suggestion ...

Reply:
Sure! I'll get that changed.

Reply:
Resolved by [8bfc6f3](https://github.com/gridap/Gridap.jl/pull/886/commits/8bfc6f3c7c7f4948843794f0d97542b4a9e28d98)